app/factories/setting.py

```diff
@@ -19,6 +19,8 @@ class Meta:
     secret = common.secret_
     # Static domain
     static_domain = common.url_
+    # Order Expiry Time
+    order_expiry_time = 15
```

Comment:
15 what?

Reply:
15 min , but i have just defined its magnitude in the defaults ? Is something wrong here?

Reply:
Write minutes in front of it, so that it doesn't confuse the reader

Reply:
It will break the build, As the type, i have specified is `Int` for the expiry time

Reply:
:roll_eyes: In comment

Reply:
Oh I understood it wrong 😅